runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpServerFactory.java

```diff
@@ -2205,7 +2218,7 @@ private void onDecodeCorsPreflight(
             assert exchange == null;
         }
 
-        private void onDecodeHeaders(
+        private DirectBuffer onDecodeHeaders(
```

Comment:
In general we don't return values from `onDecodeXXX` methods if possible.
Why not doEncodeHeaders here instead?

Reply:
Yes, returning `boolean` is better than `DirectBuffer`.